CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.0.0 (2018/12/04)
+
+Fixes:
```

Comment:
This change is not a big fix, but a change introducing a breaking change. I changed the label to “Changes” 
https://github.com/sonots/ec2-host/commit/cf9dd3bcf95bccce64d554c3c3d951ed0b40055e

Reply:
Oh,Thanks!